src/backend/routers/para.ts

```diff
@@ -29,30 +54,55 @@ export const para = router({
         first_name: z.string(),
         last_name: z.string(),
         email: z.string(),
-        role: z.string(),
       })
     )
     .mutation(async (req) => {
-      const { first_name, last_name, email, role } = req.input;
+      const { first_name, last_name, email } = req.input;
 
       const result = await req.ctx.db
         .insertInto("user")
-        .values({ first_name, last_name, email, role })
+        .values({
+          first_name,
+          last_name,
+          email: email.toLowerCase(),
+          role: "staff",
+        })
+        // TODO: make a better way to handle email conflict when adding staff that is already in db.
+        .onConflict((oc) =>
+          oc.column("email").doUpdateSet({ email: email.toLowerCase() })
```

Comment:
the update here is a no-op since by definition the email will be the same, would recommend doing `.doNothing()` instead for clarity

Reply:
If `.doNothing()` is used, then the result won't be able to generate data to return to frontend. What if I do an additional check to the db. and only create new para if the para is not there?